app/views/posts/show.html.erb

```diff
@@ -1,2 +1,8 @@
-<h1> User posts page </h1>
-<p> Details of a specific post of a user</p>
\ No newline at end of file
+<h1> User post page </h1>
```

Comment:
- [ ] Please make sure that your post show page displays post with all comments listed like  on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) as required for this project :+1: